python/hail/utils/tutorial.py

```diff
@@ -0,0 +1,164 @@
+import hail as hl
+from .java import FatalError, Env, info
+from .misc import local_path_uri, new_local_temp_dir
+import os
+import zipfile
+from urllib.request import urlretrieve
+
+__all__ = [
+    'get_1kg',
+    'get_movie_lens'
+]
+
+resources = {
+    '1kg_annotations': 'https://storage.googleapis.com/hail-tutorial/1kg_annotations.txt',
+    '1kg_matrix_table': 'https://storage.googleapis.com/hail-tutorial/1kg.vcf.bgz',
+    'movie_lens_100k': 'http://files.grouplens.org/datasets/movielens/ml-100k.zip',
+}
+
+tmp_dir: str = None
+
+
+def init_temp_dir():
+    global tmp_dir
+    if tmp_dir is None:
+        tmp_dir = new_local_temp_dir()
+
+
+def _mkdir(jhc, path):
+    if not Env.jutils().dirExists(jhc, path):
+        r = Env.jutils().mkdir(jhc, path)
+        if not r:
+            raise IOError(f'could not mkdir f{path}')
+
+
+def get_1kg(output_dir, overwrite: bool = False):
+    """Download public 1000 genotype data and sample annotations.
+
+    Parameters
+    ----------
+    output_dir
+        Directory in which to write data.
+    overwrite
+        If ``True``, overwrite existing files/directories at those locations.
```

Comment:
"overwrite any existing files/directories found at the corresponding locations."